docker-compose.yml

```diff
@@ -46,10 +50,13 @@ services:
                 scl_python_version: rh-python36
         depends_on:
             - elasticsearch
-            - elasticsearch7
+            - elasticsearch2
             - postgres
         stdin_open: true
         tty: true
+        environment:
+            ES7_HOST: http://elasticsearch:9200
+            ES_HOST: http://elasticsearch2
```

Comment:
Why are these needed now but weren't before? Why doesn't `ES_HOST` need a port?

Reply:
The configuration in our settings assumes the ES Port for Elasticsearch2 is 9200, for now I'm passing the port along with ES7 as that is how the django-elasticsearch-plugin by default expects its host to be set.